kubernetes/AKS/pod.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: model-store-pod
+spec:
```

Comment:
Please consider moving all the yaml template files to a `template` folder.

Also consider renaming pod.yaml to something more contextual